pycrostates/cluster/_base.py

```diff
@@ -168,12 +168,28 @@ def _check_fit(self):
         assert self._fitted_data is not None
         assert self._labels_ is not None
 
+    def _check_picks(self, inst, picks):
+        inst_ = inst.copy()
+        inst_.pick(picks)
+        if len(inst_.get_channel_types(unique=True)) != 1:
+            ch_types = inst_.get_channel_types(unique=False)
+            ch_types, counts = np.unique(ch_types, return_counts=True)
```

Comment:
```suggestion
    @staticmethod
    def _check_picks(info, picks):
        info = pick_info(info, picks, copy=True)
        if len(info.get_channel_types(unique=True)) != 1:
            ch_types = info.get_channel_types(unique=False)
            ch_types, counts = np.unique(ch_types, return_counts=True)
```

Let's work on the info instead of copying an entire data instance.

Also, this method does not interact with anything else than its argument (thus is static) and is only used in this class. If it was used elsewhere, I would suggest moving it as a separate function to `utils/_checks.py`, but as it's not, let's at least make it a `staticmethod`.